src/videoserver/apps/projects/routes.py

```diff
@@ -365,33 +354,10 @@ def schema_edit(self):
                 'required': False
             },
             'crop': {
-                'type': 'dict',
+                'type': 'string',
                 'required': False,
-                'empty': True,
-                'schema': {
-                    'width': {
-                        'type': 'integer',
-                        'min': app.config.get('MIN_VIDEO_WIDTH'),
-                        'max': app.config.get('MAX_VIDEO_WIDTH'),
-                        'required': True
-                    },
-                    'height': {
-                        'type': 'integer',
-                        'min': app.config.get('MIN_VIDEO_HEIGHT'),
-                        'max': app.config.get('MAX_VIDEO_HEIGHT'),
-                        'required': True
-                    },
-                    'x': {
-                        'type': 'integer',
-                        'required': True,
-                        'min': 0
-                    },
-                    'y': {
-                        'type': 'integer',
-                        'required': True,
-                        'min': 0
-                    }
-                }
+                'regex': '^\d+,\d+,\d+,\d+$'
```

Comment:
It seems like we don't have a min/max validation anymore.
I think you can do it in regex or better to create a custom validation rule
https://github.com/superdesk/video-server/blob/master/src/videoserver/lib/validator.py
https://docs.python-cerberus.org/en/stable/customize.html#custom-rules